app/i18n/messages/fi.json

```diff
@@ -194,6 +194,7 @@
   "Navbar.logout": "Kirjaudu ulos",
   "Navbar.manageReservations": " Varausten hallinta",
   "Navbar.search": "Haku",
+  "Navbar.usedLoginMethod": "Logged in with {loginMethod} __finnish__",
```

Comment:
```suggestion
  "Navbar.usedLoginMethod": "Kirjautunut sisään käyttämällä {loginMethod}",
```

Reply:
Customer wanted it to be `Kirjautumistapa`. So added that now.